test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -476,10 +468,12 @@ describe("UserWallet - Lightning Pay", () => {
             voided: false,
           }
           const count = await Transaction.countDocuments(query)
-          const { is_canceled } = await getInvoice({ lnd: lndOutside1, id })
-          return is_canceled && count === 0
+          return count === 0
```

Comment:
What are we doing with the value of the return? Where's it being used?

Reply:
this is to make sure we're in sync with the lnd/backend. the goal is to avoid having some long sleeps.

Reply:
What I meant was where is the return value being used?